src/model/engines/pep8.nit

```diff
@@ -0,0 +1,140 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pep/8 terminal engine for submissions
+#
+# TODO: Pep8Term is crap software to install and to execute.
+# Currently, this program assume that it is installed in the subdirectory `pep8term`.
+# Use `make pep8term` to install and compile it.
+module pep8
+
+import engine_base
+
+# The absolute path of a `file` in pep8term.
+# Aborts if not found
+fun pep8term(file: String): String do
+	var dir = "pep8term"
+	if not dir.file_exists then
+		print_error "{dir}: does not exists. Please install it (make pep8term)."
+		exit 1
+	end
+	file = dir.realpath / file
+	if not file.file_exists then
+		print_error "{file}: does not exists. Please check the pep8term installation."
+		exit 1
+	end
+	return file
+end
+
+# Handler for a Pep/8 submission
+class Pep8Engine
+	super Engine
+
+	redef fun language do return "Pep/8"
+
+	redef fun compile(program)
+	do
+		var player = program.player
+		var mission = program.mission
+		var source = program.source
+
+		# Get a workspace
+		# We need to create a unique working directory
+		# The following is not thread/process safe
+		var ws
+		var z = 0
+		loop
+			var date = (new TimeT).to_i.to_s + "_" + z.to_s
+			ws = "out/{date}"
+			if not ws.file_exists then break
+			z += 1
+		end
+		ws.mkdir
+		program.workspace = ws
+		print "{player}/{mission} compiled in {ws}"
```

Comment:
We should always use the `logger` from config. I like my tools silent without redirecting everything to `/dev/null`